array_api_tests/meta_tests/test_hypothesis_helpers.py

```diff
@@ -54,10 +54,11 @@ def test_kwargs():
     results = []
 
     @given(hh.kwargs(n=st.integers(0, 10), c=st.from_regex("[a-f]")))
+    @settings(max_examples=100)
```

Comment:
Yep `@settings` takes precedence over everything else. I do this so this test should pass even when we use `--max-examples=1`. This is actually what a few tests in Hypothesis does, to prevent false-negatives.

Reply:
I don't understand why you would ever pass max-examples=1. And even if you did, why wouldn't want it to apply to this test? I typically use max-examples to increase the number of examples to do a more rigorous test run.

Reply:
Hmm, a few times now I've run `pytest --max-examples=1` for the whole suite, so I thought forcing `max_examples` here would reduce noise for myself and generally prevent any noisy false-negatives if someone else had the same idea. Generally I think it clarifies that this test requires multiple runs. I imagine both of these reasons are why Hypothesis does this internally too.

Reply:
How this `test_kwargs` test method tries to find that certain things will be generated is essentially what `find_any` does in their own test suite.